### PR TITLE
go to wiki addition

### LINKED ISSUE #73: go to wiki addition
**Is your feature request related to a problem? Please describe.**

Add a line at the end of the Readme , ```Go the the Wiki for more info```
and give this like as [Click Here](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/wiki)

### PR DESCRIPTION
**Pull request by Anjali041**
Fixes #73

- [ Added line at the end of Readme to go to wiki]

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ To make a completely open sourced in house hotel management system
 
 
 -   [Complete Intro to React, v5](https://frontendmasters.com/courses/complete-react-v5/)
-    
+
```
Comment (ayan-biswas0412) [inline README.md:75]: you have added unnecessary spaces please remove them

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ I hope you are all staying safe at your home and enjoying the contribution for y
 
 ```javascript
 /**
- * 
+ *
```
Comment (ayan-biswas0412) [inline README.md:92]: here also

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Anjali041) [reply README.md:75]: yeah sure

**Review by ayan-biswas0412 — APPROVED** [review]
nice work

**Pull request merged**